lib/graphql/query/serial_execution/field_resolution.rb

```diff
@@ -35,6 +35,7 @@ def result
         def get_finished_value(raw_value)
           if raw_value.is_a?(GraphQL::ExecutionError)
             raw_value.ast_node = irep_node.ast_node
+            raw_value.path = irep_node.path
```

Comment:
😭  sadly it's totally _not_ available. 

Here's the `.map` where the index would come from: https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/query/serial_execution/value_resolution.rb#L48

This feature is a hard requirement for `DeferredExecution` and you can see the implementation here: https://github.com/rmosolgo/graphql-ruby/blob/8e7700c1047722e753f662e8740087b6ec59f3ef/lib/graphql/execution/deferred_execution.rb#L386-L391

Although ... it doesn't really apply here :S I guess we have our pick among so-so options: 

- mutate the `irep_node` for each loop in the `map` 
- copy the `irep_node` and give the copy a new `path` for each loop in the map (`irep_node.dup` _is_ supported -- I think it's used for fragment inlining)
- add some kind of proxy object to wrap the irep node & send it into each loop of the map 
- ????